data/blog/webview2.md

```diff
@@ -0,0 +1,156 @@
+
+Node.js is integrated into Electron.
+Electron applications may use any Node.js API, module, or node-native-addon from the render and browser processes.
+A WebView2 application does not assume which language or framework the rest of your application is written in.
+There are C++ and C# templates available, but in each case your JavaScript code must proxy any operating system access through the application-host process.
+
+Electron strives to maintain compatibility with the web API, including APIs developed from the [Fugu Project](https://fugu-tracker.web.app/).
+We have a [snapshot of Electron’s Fugu API compatibility](https://docs.google.com/spreadsheets/d/1APQalp8HCa-lXVOqyul369G-wjM2RcojMujgi67YaoE/edit?usp=sharing).
+WebView2 maintains a similar list of [API differences from Edge](https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/browser-features).
+
+Electron has a configurable security model for web content, from full-access to full-sandbox.
+WebView2 content is always sandboxed.
+Electron has [comprehensive security documentation](https://www.electronjs.org/docs/tutorial/security) on choosing your security model.
+WebView2 also has [security best practices](https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/security).
+
+The Electron source is maintained and available on GitHub.
+Applications can modify can build their own _brands_ of Electron.
+The WebView2 source is not available on GitHub.
+
+Quick Summary:
+
+|                                           | Electron        | WebView2                     |
+| ----------------------------------------- | --------------- | ---------------------------- |
+| Build Dependency                          | Chromium        | Edge                         |
+| Source Available on GitHub                | Yes             | No                           |
+| Shares Edge/Chrome DLLs                   | No              | No                           |
+| Shared Runtime Between Applications       | No              | Optional                     |
+| Application APIs                          | Yes             | No                           |
+| Node.js                                   | Yes             | No                           |
+| Sandbox                                   | Optional        | Always                       |
+| Requires a Separate Application Framework | No              | Yes                          |
+| Supported Platforms                       | Mac, Win, Linux | Win (Mac on roadmap)         |
+
+## Performance Discussion
+
+Now the big question:
+_Will your app use 50% less memory by switching between one embedding of Chromium and another?_
+Unfortunately not, but that would have been rad.
+
+We created the [scaffolding to compare some simple apps built using Electron, WPF+WebView2, and .NET+WebView2](https://github.com/crossplatform-dev/xplat-challenges/blob/main/results.md). 
```

Comment:
"We" is a bit ambiguous here — I know @molant was working on crossplatform.dev, but is this explicitly an Electron project? Or is it a Microsoft one?

Reply:
> is this explicitly an Electron project? Or is it a Microsoft one?

None 🙃
My hope is to create something community driven with the help of different crossplatform technologies. I agree that "we" here is weird and the scope of the project is not only WV2 and Electron.

Reply:
Resolving as "okay enough" for this post.